storybook/src/example-pages/Patronen/zoeken-in-lijst-cards.content.ts

```diff
@@ -16,6 +16,7 @@ export const cardList: CardList<TemplateResult> = {
           iconMode: "only",
         },
       ],
+      dsoCardClick() {},
```

Comment:
Zie bovenstaande opmerking over ouders en kinderen

Reply:
Is weg, want optioneel in model